workspaces/sample-workspace/client-extensions/sample-theme-spritemap-3/client-extension.yaml

```diff
@@ -0,0 +1,9 @@
+assemble:
+    - from: build
+      include: "spritemap.svg"
```

Comment:
Talking off the top of my head, but I think this value may not need quotes... :thinking: